autobot/telegram/inline/objects.py

```diff
@@ -1,6 +1,24 @@
 from ..objects import BaseObject, User, Location, MessageEntity, InlineKeyboardMarkup
 from typing import Optional
 
+
+class InputMessageContent(BaseObject):
+    """
+    This object represents the content of a message to be sent as a result of an inline query. 
+    Telegram clients currently support the following 5 types:
+    
+    `InputTextMessageContent`
+
+    `InputLocationMessageContent`
+
+    `InputVenueMessageContent`
+
+    `InputContactMessageContent`
+    
+    `InputInvoiceMessageContent`
+    """
+    pass
```

Comment:
@kallyas I think this whole class needs to be declared in the same class where the subclasses are, so that it's used to declare those subclasses. I can't even file the file in which the other classes are declared, but they need to be in the same file, then you can import that base class to other modules where it is needed

Reply:
@kallyas I've just re-read the pr and I think it makes sense. I was still trying to figure out the files it will link from but I think it makes sense to declare it here. I think I just didn't think that the inline module would have other objects that could start with something else other than inline. Should we remove the general `objects.py` file and create the two independent file? I personally think it would be great for organization purposes to separate these two

Reply:
Here's what I think. We rename ``object`` to ``query``

Then create another file and call it input which will house all InputMessageContect related object. The ``InputMessageContext`` class implements ``BaseObject`` and then the rest of the four implement ``InputMessageContext``. 

For now, the common ground is that they relate to ``InlineQuery``, so we can group them under the inline module.

Reply:
Here's my suggestion. We rename ``object`` to ``query``

Then create another file and call it input which will house all ``InputMessageContect`` related objects. The ``InputMessageContext`` class implements ``BaseObject`` and then the rest of the four implement ``InputMessageContext``. 

For now, the common ground is that they all relate to ``InlineQuery``, so we can group them under the inline module.

Reply:
> @kallyas I've just re-read the pr and I think it makes sense. I was still trying to figure out the files it will link from but I think it makes sense to declare it here. I think I just didn't think that the inline module would have other objects that could start with something else other than inline. Should we remove the general `objects.py` file and create the two independent file? I personally think it would be great for organization purposes to separate these two

That's what I looked at in the first place, It makes more sense to separate the two

Reply:
@ht-thomas @ushergodwin Any decision made yet for this?